source/includes_main/_enrollments.md

```diff
@@ -98,6 +101,7 @@ Parameter | Required? | Type | Description
 prospect_id | **yes** | *integer* | The prospect's ID
 step_id | no | *string* | The step's ID of the automation at which the prospect must be enrolled.
 reenroll | no | *boolean* | Set to `false` if you want to prevent re enroll the prospect because he already has been enrolled. **If not set to** `false` **this parameter will always be** `true`
+start_on | no | *string* | The date on which the enrollment must be started in the ISO 8601 format (YYYY-MM-DD). If not set or set in the past, the enrollment will be started immediately.
```

Comment:
[Copy] `the enrollment will be started immediately` => `the enrollment will start immediately`